x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/bulk_actions/action_to_rules_client_operation.ts

```diff
@@ -56,8 +57,10 @@ export const bulkEditActionToRulesClientOperation = (
         {
           field: 'actions',
           operation: 'add',
-          value: transformToActionFrequency(action.value.actions, action.value.throttle).map(
-            transformNormalizedRuleToAlertAction
+          value: parseAndTransformRuleActions(
```

Comment:
Would be nice to have some tests for the `add_rule_actions` and `set_rule_actions` for normal actions and system actions in `x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/bulk_actions/action_to_rules_client_operation.test.ts`

Reply:
Added here: 38481967c4774607ebd1fa75768e2fe6677ea08f